spec/identity_doc_auth/acuant/request_spec.rb

```diff
@@ -152,5 +152,58 @@
         expect(response.exception).to be_a(Faraday::ConnectionFailed)
       end
     end
+
+    context 'when the request resolves with a handled http error status' do
+      before do
+        allow(subject).to receive(:handle_http_response) do |http_response|
+          http_response
+        end
+      end
+
+      def shared_expects(response)
+        expect(response.success?).to eq(false)
+        expect(response.exception).to be_kind_of(IdentityDocAuth::RequestError)
+        expect(response.exception.message).not_to be_empty
+      end
+
+      it 'it produces a 438 error' do
+        stub_request(:get, full_url).
+          with(headers: request_headers).
+          to_return({ body: 'test response body', status: 438 })
+
+        expect(exception_notifier).not_to receive(:call)
+
+        response = subject.fetch
+
+        shared_expects(response)
+        expect(response.errors).to eq({ general: [IdentityDocAuth::Errors::IMAGE_LOAD_FAILURE] })
```

Comment:
style: i'd drop the optional braces (maybe a Rubocop setting we forgot in this repo, idk)

```suggestion
          to_return(body: 'test response body', status: 438)

        expect(exception_notifier).not_to receive(:call)

        response = subject.fetch

        shared_expects(response)
        expect(response.errors).to eq(general: [IdentityDocAuth::Errors::IMAGE_LOAD_FAILURE])
```

and same for the other examples

Reply:
gah, I thought of this and then forgot to clean it up. Thanks.